php/razorpay_js/index.html

```diff
@@ -0,0 +1,402 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Chargebee Razorpay Integeration Sample</title>
+	<style>
+		body{
+			height: 100vh;
+			overflow: hidden;
+		}
+		h1{
+			text-align: center;
+		}
+		.button-container {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			height: 50%;
+		}
+		.btn-pay{
+			cursor: pointer;
+			background: #003eff;
+			border: none;
+			padding: 12px 15px;
+			border-radius: 5px;
+			color: #e5e5e5;
+			text-transform: uppercase;
+			font-weight: 600;
+    		box-shadow: #0010ff29 2px 2px 6px;
+		}
+		.disable{
+			pointer-events: none;
+			opacity: 0.7;
+		}
+		.dropdown{
+			width: 120px;
+			height: 40px;
+			margin: 1rem;
+			padding: 0rem 1rem;
+			color: #003eff;
+			border: 2px solid #003eff;
+			border-radius: 5px;
+			font-weight: 600;
+		}
+	</style>
+</head>
+
+<body>
+	<h1>Chargebee Razorpay Integeration Sample</h1>
+	<div class="button-container">
+		<select class="dropdown"  id="payment_type" name="payment_type">
+			<option selected value="card">Card</option>
+			<option value="upi">UPI</option>
+		</select>
+		<button class="btn-pay" id="rzp-button1" onclick="initatePayment()">
+			Pay with Razorpay
+		</button>
+	</div>
+	<script src="https://checkout.razorpay.com/v1/checkout.js"></script>
+	<script>
+		let ITEM_PRICE_ID = "cbdemo_advanced-INR-Yearly"; // replace with your item price id
+		const payButton = document.getElementById("rzp-button1");
+		let chargebeeCustomerDetail;
+		let CB_CUSTOMER_ID;
+		let paymentType;
+		async function initatePayment() {
+			try {
+				paymentType = document.getElementById("payment_type").value;
+				payButton.classList.add('disable')
+				if(!chargebeeCustomerDetail) {
+					console.info(`=============== Creating Customer ================`);
+					/**
+					 * @note You can skip this create customer part is the customer aleady exists in Chargebee Application
```

Comment:
You can skip this create customer part is the customer aleady exists in Chargebee Application -> `This step can be skipped if the customer already exists within Chargebee`

Reply:
Done